README.md

```diff
@@ -33,7 +33,9 @@ to get a feel for how to use it.
 
 ## Component
 
-Once the installation is done, you'll need to generate your own component and re-export
+You have the opportunity to customize if needed.
+
+To do this, generate your own component and re-export
```

Comment:
Because the addon component is [re-exported in the `app/` folder](https://github.com/kennethkalmer/ember-cli-nouislider/blob/master/app/components/range-slider.js), it is automatically merged into the consuming app's namespace.

Therefore, the user does not have to generate the component and extend the addon unless they want to customize the behavior.